docker/Dockerfile

```diff
@@ -8,28 +8,40 @@ ARG TRAVIS
 ARG TRAVIS_BRANCH
 
 COPY flecsi-third-party/ /home/flecsi/tpl
+RUN rm -rf /home/flecsi/.ccache
 COPY ccache/ /home/flecsi/.ccache
 USER root
-RUN chown -R flecsi:flecsi /home/flecsi/tpl /home/flecsi/.ccache /usr/local
+RUN chown -R flecsi:flecsi /home/flecsi/tpl /home/flecsi/.ccache
 USER flecsi
 
 WORKDIR /home/flecsi/tpl
-RUN mkdir -p build && cd build && \
-    cmake -DBUILD_SHARED_LIBS=ON \
+RUN mkdir build
+RUN mkdir install
+WORKDIR build
+RUN ccache -z
+RUN cmake -DBUILD_SHARED_LIBS=ON \
           -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
           -DUSE_SYSTEM_LIBS=${SYSTEM_LIBS} \
-          .. && make -j2;
+          -DGASNet_ROOT_DIR=/home/flecsi/tpl/install \
+          -DCMAKE_INSTALL_PREFIX=/home/flecsi/tpl/install/ \
+          ..
+RUN make -j2
+RUN ccache -s
 
 USER root
-RUN chown -R root:root /usr/local
+RUN make install
 USER flecsi
+RUN ccache -s
```

Comment:
A second time?

Reply:
Ohh it's copied over from the dockerfile from vocta...